_projects/2023/100481838/100481838.Rmd

```diff
@@ -0,0 +1,1067 @@
+           x=-1.8, 
+           y=-0.02, 
+           label= "Cold", 
+           color = "#54a5ba",
+           fontface = 2)+
+  annotate("text", 
+           x=1.8, 
+           y=-0.02, 
+           label= "Hot", 
+           color = "#eb8550",
+           fontface = 2)+
+  annotate("text", 
+           x=5, 
+           y=-0.02, 
+           label= "Extremely Hot", 
+           color = "#cc2821",
+           fontface = 2)
+
+plot1980.1990
+```
+
+Once that is done, we can add a few more annotations are needed. Namely those that are placed together with an arrow or another visual element. For this we use the function `geom_segment` and indicate its positioning with x y coordinates as we have done for text annotations.
+
+```{r}
+plot1980.1990 <- 
+  plot1980.1990 +
+  annotate("text", 
+           x=3, 
+           y= 0.35, 
+           label= "1980-1990", 
+           color = "black",
+           size = 10,
+           fontface = 10)+
+  geom_segment(
+    x = -4.2, 
+    y = 0.05,          
+    xend = -4.2, 
+    yend = 0.1,
+    color = "#999999",
+    arrow = arrow(length = unit(0.2, "cm"), 
+                  ends = "last", 
+                  type = "closed"))+
+  annotate("text", 
+           x= -4.2, 
+           y= 0.03, 
+           label= "More frequent", 
+           color = "#999999")
+
+plot1980.1990  
+```
+
+Once we have that down, we can do it for all the other periods.
+
+```{r all the periods, echo=FALSE, fig.height=12, fig.width=12}
+#Period of 1991-2001
+plot1991.2001<-
+longdata |> 
+  filter(period == "1991.2001") |> 
+  ggplot(aes(x = anoms.S, 
+               y = value, 
+               fill = temptype)) + 
+    geom_area(stat = "identity", 
+                   position = "stack", 
+                   bins = 30,
+                   show.legend = FALSE) +
+    scale_fill_manual(values = c("Extremely cold"="#085b84", 
+                                 "Cold" = "#54a5ba",
+                                 "Normal" = "#dfdfdf", 
+                                 "Hot" = "#eb8550",
+                                 "Extremely hot" = "#cc2821")) +
+    xlim(c(-6, 6))+
+    theme_void()+
+  labs(title = "Summer temperatures",
+       subtitle = "in the Northern Hemisphere") +
+  theme(plot.title = element_text(color = "black",
+                                  hjust = 0.015,
+                                  face = "bold"),
+        plot.subtitle = element_text(color = "gray",
+                                     hjust = 0.015))+
+  geom_vline(xintercept = c(-3, -0.5, 0.5, 3), 
+             linetype = "dashed", 
+             color = "gray")+ 
+    annotate("text", 
+           x=0, 
+           y=-0.02, 
+           label= "Normal", 
+           color = "#999999",
+           fontface = 2)+
+  annotate("text", 
+           x=-5, 
+           y=-0.02, 
+           label= "Extremely Cold", 
+           color = "#085b84",
+           fontface = 2)+
+  annotate("text", 
+           x=-1.8, 
+           y=-0.02, 
```

Comment:
I see that there is a lot of repeated code throughout the post, which makes the whole thing harder to read. To avoid this, you could encapsulate those things in reusable functions, something like:

```r
custom_annotations <- function(plot) {
  plot + annotate() + annotate() + annotate()
}
```

and then reuse it.